cli/src/clojure_lsp/server.clj

```diff
@@ -205,8 +213,6 @@
         timbre-logger (doto (->TimbreLogger db*)
                         (logger/setup))
         _ (logger/info "[SERVER]" "Starting server...")
-        is (or System/in (lsp/tee-system-in System/in))
-        os (or System/out (lsp/tee-system-out System/out))
```

Comment:
I can't recall why we have tee-system-in, if that is not needed indeed, we could remove from lsp4clj in the future